crt_portal/cts_forms/views.py

```diff
@@ -344,19 +342,15 @@ class ProfileView(LoginRequiredMixin, FormView):
 
     def post(self, request):
         """Update or create Profile"""
-        user = request.user
-        intake_filter = request.POST.get('intake_filters')
-
-        if intake_filter:
-            instance = get_object_or_404(Profile, id=user.id)
+        if request.user.profile:
```

Comment:
We'll need to handle this situation a bit more carefully. As written, this would never evaluate to `False`. It would throw an `RelatedObjectDoesNotExist` exception if no profile existed.